stock_picking_inter_warehouse/views/stock_warehouse_views.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_warehouse" model="ir.ui.view">
+        <field name="name">stock.warehouse</field>
+        <field name="model">stock.warehouse</field>
+        <field name="inherit_id" ref="stock.view_warehouse" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='delivery_steps']/.." position="inside">
+                <newline />
+                <field
+                    name="inter_warehouse_transfers"
+                    attrs="{'invisible':[('id', '=', False)]}"
+                />
+                <newline />
+                <field
```

Comment:
Those fields should go into the configuration page.

Moreover those fields should not be required as the write() in stock.warehouse won't never be called.

Reply:
Hi @rousseldenis , help me understand on a functional level your request: 

To enable one warehouse to receive inter-warehouse transfers, each one of these fields must be populated:

![image](https://github.com/OCA/stock-logistics-workflow/assets/55489873/5a3c2f2c-428b-4677-8091-443f9049002e)

By using the flag, we're enabling inter-WH feature only for relevant WHs. What would be the benefit in changing this behavior?

Reply:
In the implementation you tried to mimic the Odoo warehouse configuration like the delivery/reception steps confguration in that code : https://github.com/OCA/stock-logistics-workflow/pull/1383/commits/70529d4b26b966862a7a1fe3192133a0631dabec#diff-e2cd4f64afccac0554171922b36a7863041339acea639a2ba3dc6b9fb96079c0R20

But, ok, you rely on those fields in order to create the rule (or update it).

In addition, I would have added a good help message for those fields.